### PR TITLE
Garage vectorized sampler and other small edits. 

### PR DESCRIPTION
**Pull request by akashvelu**
- Vectorized worker sampler for garage. 
- Add optimized one segment dendrite. 
- Fix metaworld install in requirements.txt
- Variable renames.

### CODE REVIEW

File: nupic/embodied/models/multiheaded_dendrite_mlp.py [hunk 1]
```diff
@@ -84,16 +86,16 @@ def __init__(self,
         self.k_winners = k_winners
         self.relu = relu
```
Comment (lucasosouza) [inline nupic/embodied/models/multiheaded_dendrite_mlp.py:87]: self.relu is not being used

File: nupic/embodied/samplers/single_vecworker_sampler.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        worker_args (dict or None): Additional arguments that should be passed
+            to the worker.
+
+    """
+
+    def __init__(
+            self,
+            agents,
+            envs,
+            *,  # After this require passing by keyword.
```
Comment (lucasosouza) [inline nupic/embodied/samplers/single_vecworker_sampler.py:67]: Why the asterisk, is that required?

File: nupic/embodied/samplers/single_vecworker_sampler.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            worker_factory=None,
+            max_episode_length=None,
+            is_tf_worker=False,
+            seed=SEED,
```
Comment (lucasosouza) [inline nupic/embodied/samplers/single_vecworker_sampler.py:71]: SEED is global, but not a constant. The all caps standard in python is used for constants.

File: requirements.txt [hunk 4]
```diff
@@ -63,7 +63,7 @@ stable_baselines3
 colorama==0.4.3
 urllib3==1.25.11
 
-git+https://github.com/rlworkgroup/metaworld.git@master#egg=metaworld
+git+https://github.com/rlworkgroup/metaworld.git@d9a75c451a15b0ba39d8b7a8b6d18d883b8655d8#egg=metaworld
```
Comment (lucasosouza) [inline requirements.txt:66]: It is not an option to fix a pip version for them instead of a SHA?

**Review by lucasosouza — APPROVED** [review]
Looks good, left small comments and questions.

File: requirements.txt (hunk 4 above)
Comment (akashvelu) [reply requirements.txt:66]: No, unfortunately they don't have pip releases yet.

File: nupic/embodied/samplers/single_vecworker_sampler.py (hunk 2 above)
Comment (akashvelu) [reply nupic/embodied/samplers/single_vecworker_sampler.py:67]: This is how this was done in the garage sampler files, so I stayed consistent with their code.

File: nupic/embodied/models/multiheaded_dendrite_mlp.py (hunk 1 above)
Comment (akashvelu) [reply nupic/embodied/models/multiheaded_dendrite_mlp.py]: Fixed

**Pull request merged**